### ISSUE TITLE
[Redesign] Add the possibility to also enter Recommendations/Tips

### ISSUE THREAD

**Issue by rianrietveld** [w3c/wcag-em-report-tool#389]
Add the possibility to give more feedback in the report. For example about issues that are not covered by the WCAG level testet for, but have an impact on the accessibility of the site. 

Like remarks about heading structure or semantics. But also to report bugs in the software or give compliments to the developers for work nicely done.

I add such recommendations with all my reviews and most of the time these issues aslo end up on the items-to-be-fixed list.

**Comment by hidde** [comment]
Related to #247 #245

**Comment by nitedog** [comment]
This feature partially exists already:

- You can add recommendations in the "Observations"
- You can provide "Observations" (and/or "Results") for Success Criteria beyond the target conformance level

Adding additional recommendations could be confused with formal requirements and would need to be presented carefully.

**Comment by hidde** [comment]
> This feature partially exists already:
> *  You can add recommendations in the "Observations"
> * You can provide "Observations" (and/or "Results") for Success Criteria beyond the target conformance level

I think some reviewers like to split their recommendations between:

* observations that constitute a normative WCAG failure
* general comments

If they go in the same box, the idea below would be harder to execute, both for us on our Report page, and for people who export the JSON and use it in their own tooling.

> Adding additional recommendations could be confused with formal requirements and would need to be presented carefully.

In reports from various organisations I've seen these get explicitly split out under separate headings. One heading for all WCAG failures and conformance observations, and one for non-normative comments.

For instance, under “Issues” there are things like “1.1.1 failure: this image has no alternative text”, under “Tips” are things like “This alternative text is accurate, but a bit wordy, maybe remove rephrase as … …”.

**Comment by TalyahAtUXAccess** [comment]
I agree with this request. The observation area is not enough to gather all the information we as auditors need to convey. In my reports this area includes an "How to Fix" area. In this area, I have excerpts from the current code followed by the same code with the fixes. This content is presented in a different font highlighting the changes that I have made. I can't do it here. 
If I want to add semantics like a heading for example, I can't do it with the current interface of the reporting tool.

**Comment by hidde** [comment]
Hi @TalyahAtUXAccess! What you're describing will be possible in the updated version of this tool.

Preview: http://wcag-em-report-tool-2021-redesign.netlify.app

Repository for bug reports etc https://github.com/w3c/wai-wcag-em-report-tool

**Comment by hidde** [comment]
Here's a demo of what it could look like if we did per issue reporting: https://codepen.io/hidde/pen/gOwmvpo
